p8e-shared/src/main/kotlin/io/provenance/p8e/shared/domain/Envelope.kt

```diff
@@ -72,6 +72,10 @@ open class EnvelopeEntityClass : UUIDEntityClass<EnvelopeRecord>(
         EnvelopeTable.groupUuid eq groupUuid and (EnvelopeTable.executionUuid eq executionUuid)
     }.toList()
 
+    fun findByExecutionUuid(executionUuid: UUID) = find {
```

Comment:
Shouldn't we find with public key? for PII purposes.

Reply:
No, because we need to get all parties envelopes for this execution that exist on a single p8e instance, so they can all proceed with being completed (after we index the scope)